ZelBack/src/services/fluxCommunication.js

```diff
@@ -12,6 +13,7 @@ const log = require('../lib/log');
 const serviceHelper = require('./serviceHelper');
 const daemonService = require('./daemonService');
 const benchmarkService = require('./benchmarkService');
+const appsService = require('./appsService');
```

Comment:
That needs to get reverted as appsService is referencing fluxCommunication, this can result in bad loop.

Reply:
Commit done to "fix" it on developmentBeta